q2_moshpit/citations.bib

```diff
@@ -129,3 +129,20 @@ @misc{NCBI
   url = {https://www.ncbi.nlm.nih.gov/},
   note = {Bethesda (MD): National Library of Medicine (US), National Center for Biotechnology Information;},
 }
+
+@article{huerta-cepas_eggnog_2019,
+	title = {{eggNOG} 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses},
+	volume = {47},
+	issn = {0305-1048},
+	shorttitle = {{eggNOG} 5.0},
+	url = {https://doi.org/10.1093/nar/gky1085},
+	doi = {10.1093/nar/gky1085},
+	abstract = {eggNOG is a public database of orthology relationships, gene evolutionary histories and functional annotations. Here, we present version 5.0, featuring a major update of the underlying genome sets, which have been expanded to 4445 representative bacteria and 168 archaea derived from 25 038 genomes, as well as 477 eukaryotic organisms and 2502 viral proteomes that were selected for diversity and filtered by genome quality. In total, 4.4M orthologous groups (OGs) distributed across 379 taxonomic levels were computed together with their associated sequence alignments, phylogenies, HMM models and functional descriptors. Precomputed evolutionary analysis provides fine-grained resolution of duplication/speciation events within each OG. Our benchmarks show that, despite doubling the amount of genomes, the quality of orthology assignments and functional annotations (80\% coverage) has persisted without significant changes across this update. Finally, we improved eggNOG online services for fast functional annotation and orthology prediction of custom genomics or metagenomics datasets. All precomputed data are publicly available for downloading or via API queries at http://eggnog.embl.de},
```

Comment:
Let's remove the abstract from here - one can always just look it up online. In fact, could you please remove it from all of the entries that happen to have it here? Thanks!